docs/dependency_confusion.md

```diff
@@ -19,127 +18,128 @@ of the internal one.
+To the system that runs Cachi2, nothing. Cachi2 never executes your code, or the code of any of your
+dependencies. What about your build?
 
-You are likely using Cachito through OSBS. Cachito will serve the malicious dependency to the OSBS
-build, where you will subsequently use it to build your application container. The malicious
-behaviour may or may not manifest at build time. In the best case scenario, the build will fail.
-In the worst case scenario, you will end up shipping malicious code to customers.
+Hopefully, you are building your application in an isolated environment (such as a container build
+with no network access) to protect your own system. In the best case scenario, the malicious behaviour
+will manifest at build time and the build will fail. In the worst case scenario, the build will succeed
+but the built product will contain malicious code, which may then affect whoever tries to run it.
 
-## Dependency Confusion vs Cachito
+## Dependency Confusion vs Cachi2
 
 Before delving into the specifics of each supported package manager, let us make a general statement
-about Cachito’s stance towards dependency confusion.
+about Cachi2’s stance towards dependency confusion.
 
-Cachito aims to match the behaviour of your package manager as closely as possible. Generally
-speaking, Cachito is as vulnerable to dependency confusion as your package manager, not more, not
+Cachi2 aims to match the behaviour of your package manager as closely as possible. Generally
+speaking, Cachi2 is as vulnerable to dependency confusion as your package manager, not more, not
 less. However, all the supported package managers provide means to protect yourself by following
 best practices. For example, you can remove all known attack vectors by simply verifying the
 checksums of your dependencies.
 
-Cachito enforces *some* of these best practices. Typically, each package manager provides a lockfile
-or other mechanism for pinning dependency versions. Cachito requires this. Verifying checksums is
+Cachi2 enforces *some* of these best practices. Typically, each package manager provides a lockfile
+or other mechanism for pinning dependency versions. Cachi2 requires this. Verifying checksums is
 usually up to you.
 
-Additionally, due to the caching nature of Cachito, you should never be affected by dependency
-confusion during a *rebuild*. If you build the same revision of your repository, or a different
-revision which uses the same versions of dependencies, Cachito will use the dependencies that it has
-already downloaded. Dependency confusion should only be a concern when building for the first time
-or when changing the version of a dependency.
-
 # Package Managers
 
-*Sometimes users may state that Cachito supports a certain programming language. This is not
-entirely accurate. Cachito only supports package managers. The ecosystem of a programming language
+*Sometimes users may state that Cachi2 supports a certain programming language. This is not
```

Comment:
It doesn't hurt to leave this here, but I wonder if this disclaimer is needed at all in this doc.

Reply:
True, I'll remove this